pkgs/top-level/php-packages.nix

```diff
@@ -199,6 +199,8 @@ lib.makeScope pkgs.newScope (self: with self; {
 
     phpstan = callPackage ../development/php-packages/phpstan { };
 
+    phpunit = callPackage ../development/php-packages/phpunit { };
```

Comment:
This should not be in `pkgs/top-level/php-packages.nix` . You should consider `phpunit` just like an arbitrary binary.